### DISCUSSION TITLE
HID keyboard function implementation

### DISCUSSION THREAD

**Discussion by eliwss0** [InfiniTimeOrg/InfiniTime#1604] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

HID keystroke functions to send to smartphone

### Description

I've been working with the HID branch with the goal of implementing an HID keystroke to trigger the camera shutter remotely on Android phones, but I think there could be useful implementations outside of this one use case.

To make implementation easier, I think a collection of simple functions to manage the HID service and send a keystroke will help facilitate higher level programming. This will make it easier to implement in other features that require HID communication.

**Comment by earboxer** [comment]
I have this dream of covertly receiving messages as morse code vibrations, and being able to reply by tapping out morse code sequences, so maybe the second part of this would go along with the "useful implementations outside of this one use case" idea...
